com.paledega.alpinelinux.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.github.alpine-linux
```

Comment:
this still doesn't meet the requirements.

Reply:
manifest file renamed.

Reply:
Is com.github.paledega.alpine-rootfs valid?

Reply:
See the documentation

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
New app-id: io.github.paledega.alpine-rootfs